services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:
```

Comment:
The return type hint for `get_cinema_halls` should be `QuerySet` instead of `CinemaHall` because the function returns a QuerySet of all `CinemaHall` instances, not a single instance.